publish/2020-05-07-EJB-persistent-timers.adoc

```diff
@@ -0,0 +1,224 @@
+---
+layout: post
+title: "Updates to EJB persistent timers coordination and failover across members is now available on Open Liberty 20.0.0.5"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/tomjenningss
+author_github: https://github.com/tomjenningss
+seo-title: Open Liberty 20.0.0.5 allows developers to add a configurable attribute to an EJB Persistent Timer feature on Open Liberty 20.0.0.5.
+seo-description: Open Liberty 20.0.0.5 allows developers to add a configurable attribute to an EJB Persistent Timer feature.
+blog_description: Open Liberty 20.0.0.5 allows developers to add a configurable attribute to an EJB Persistent Timer feature.
+---
+= Updates to EJB persistent timers coordination and failover across members is now available on Open Liberty 20.0.0.5
+
+Tom Jennings <https://github.com/tomjenningss>
+:imagesdir: /
+:url-prefix:
+:url-about: /about/
+
+// tag::intro[]
+With Open Liberty 20.0.0.5, developers can add a configurable attribute to the EJB Persistent Timer feature. The new attribute sets a maximum amount of time allowed for a persistent timer to complete before another server can take over and run the timer instead.
+
+In link:{url-about}[Open Liberty] 20.0.0.5:
+
+* <<EPT, EJB Persistent Timers>>
+* <<CLM, JAAS Custom Login Modules>>
+* <<OLL, Open Liberty Logging>>
+* <<JLO, Omit Specified Fields From JSON Logging Output>>
+
+View the list of fixed bugs in https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A20005+label%3A%22release+bug%22+
```

Comment:
I missed this in my other review (sorry!), but should we have link text here rather than just the link with all the filter info. It looks like past releases it's been "View the list of fixed bugs in [20.0.0.5](https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A20005+label%3A%22release+bug%22+)."